### ISSUE TITLE
Documentation link broken

### ISSUE THREAD

**Issue by centurianii** [bitovi/jquerypp#133]
From http://jquerypp.com trying to find documentation for drag or swipe.

Also download builder link is broken (look down at `Why jQuery++ -> Flexible`).

**Comment by justinbmeyer** [comment]
Thanks for reporting.  jQuery++ isn't actively maintained right now.  I created a proposal for how to make these things useful again in a more modern development workflow: https://github.com/donejs/donejs/issues/1117

This should also keep them useful even if they aren't being actively maintained.

**Comment by centurianii** [comment]
I use drag with excellent results in desktop & Android for some projects. I think it's a very good companion for jQuery!

**Comment by justinbmeyer** [comment]
Yes, all of canjs’s events can be exported to jQuery for standalone use. So this would still work.
